response.cc

```diff
@@ -61,3 +104,110 @@ std::string Response::ToString() const
 
 	return ss.str();
 }
+
+Response::ResponseCode Response::IntToResponseCode(int code) {
+	switch(code) {
```

Comment:
Nope; this safely casts values from int to the enum (you can't safely casts arbitrary ints otherwise). It's effectively a getter.